content/03.methods.md

```diff
@@ -5,8 +5,59 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+We used simple pythonic functions called label functions to quickly annotate our candidates sentences.
```

Comment:
Start by bringing your reader into the idea of label functions in the context of the snorkel-style approach.
1-2 sentences introducing the broad idea of Snorkel's approach.
1 sentence on conceptually what label functions do within that context.
1 sentence on conceptually how label functions are combined (note the generative/discriminative model sections coming later).

Then start.